lametro/utils.py

```diff
@@ -106,3 +109,38 @@ def get_list_from_csv(filename):
         my_list = [row for row in reader]
 
     return my_list
+
+
+class LAMetroRequestTimeoutException(Exception):
+    pass
+
+
+def get_url(url, params=None, **kwargs):
```

Comment:
How about something a bit more descriptive? `timed_get`? Open to ideas!

Reply:
`timed_get` works for me!